table.scss

```diff
@@ -0,0 +1,126 @@
+@import 'bower_components/vui-colors/colors.scss';
+@import 'bower_components/vui-typography/typography.scss';
+@import 'bower_components/vui-typography/px-to-base-rem.scss';
+
+@mixin vui-table() {
+	background-color: #ffffff;
+	border: 1px solid $vui-color-titanius;
+	border-collapse: separate;
+	border-radius: 6px;
```

Comment:
When I was talking with Dana, he mentioned the corners should be rounded 6px, do we want the corner rounding to be variable?